docs/standard/modern-web-apps-azure-architecture/develop-asp-net-core-mvc-apps.md

```diff
@@ -5,6 +5,7 @@ author: ardalis
 ms.author: wiwagn
 ms.date: 06/28/2018
 ---
+
```

Comment:
Although it's rendering identically, the added line has evidently produced a build warning. Please remove it.

Reply:
Done